src/main/java/se/sundsvall/supportmanagement/integration/db/util/ConfigPropertyExtractor.java

```diff
@@ -0,0 +1,66 @@
+package se.sundsvall.supportmanagement.integration.db.util;
+
+import static java.util.Objects.isNull;
+import static java.util.Optional.ofNullable;
+import static org.zalando.problem.Status.NOT_FOUND;
+
+import org.apache.commons.lang3.Strings;
+import org.zalando.problem.Problem;
+import se.sundsvall.supportmanagement.integration.db.model.NamespaceConfigEntity;
+import se.sundsvall.supportmanagement.integration.db.model.NamespaceConfigValueEmbeddable;
+
+public class ConfigPropertyExtractor {
+	private ConfigPropertyExtractor() {}
+
+	public static final String PROPERTY_DISPLAY_NAME = "DISPLAY_NAME";
+	public static final String PROPERTY_SHORT_CODE = "SHORT_CODE";
+	public static final String PROPERTY_NOTIFICATION_TTL_IN_DAYS = "NOTIFICATION_TTL_IN_DAYS";
+	public static final String PROPERTY_ACCESS_CONTROL = "ACCESS_CONTROL";
+	public static final String PROPERTY_NOTIFY_REPORTER = "NOTIFY_REPORTER";
+
+	/**
+	 * Get the value for provided key as the type that is defined for the key/value-pair or null if no property matching
+	 * provided key is found.
+	 *
+	 * @param  nullableNamespaceConfigEntity config entity to find property value in
+	 * @param  key                           value of key to match
+	 * @return                               value for property whos key is matching sent in key or null if no match is
+	 *                                       found
+	 */
+	@SuppressWarnings("unchecked")
+	public static <T extends Object> T getOptionalValue(NamespaceConfigEntity nullableNamespaceConfigEntity, String key) {
```

Comment:
The name implies that an Optional is returned, but it isn't.

Since there isn't any Optionals involved, why not just name the method: `getValue()` ?